package.json

```diff
@@ -87,8 +87,9 @@
     "unistore": "^3.5.2",
     "uslug": "^1.0.4",
     "web-vitals": "^3.0.4",
-    "webdev-infra": "1.0.42",
-    "xss": "^1.0.14"
+    "webdev-infra": "1.0.44",
+    "xss": "^1.0.14",
+    "@mdn/browser-compat-data": "5.2.33"
```

Comment:
Probably a silly question - but I expected `@mdn/browser-compat-data` to already exist as a dependency, so double checking if this WAI.

Reply:
Discussed offline.

Reply:
It is a dependency of webdev-infra, with a version range. To make dependabot pick it up it needs to be specified in the main package.json as well.